man/org.freedesktop.systemd1.xml

```diff
@@ -2853,6 +2853,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly aay LogExtraFields = [[...], ...];
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s LogFilterRegex = '...';
```

Comment:
hmm, a simple string? should be an array of strings, and a boolean indicating allowlist/denylist